assets/sass/components/global/_googlesitekit-cta-link.scss

```diff
@@ -127,8 +151,18 @@ button {
 	&.googlesitekit-cta-link {
 		color: $c-content-primary;
 
+		/* stylelint-disable-next-line no-descending-specificity */
```

Comment:
Why is this being disabled?

If it makes sense to do so here, we should add a comment explaining why. Similar to `!important` flags—if it needs to be done that's okay, but we should leave context for future developers reading the code 😄

Reply:
I added comments. Didn't see it in other places, so I thought it is not necessary.

Reply:
But why are you forcing the rule to be disabled? Does it break something with the selectors, etc.? We have the rules in place so the code is consistent, but if the rule is causing bugs that's worth highlighting—maybe we'll want to reconsider the rule then.

Why can't you change the code to follow the rule here? What would it look like with this `/* stylelint-disable-next-line no-descending-specificity */` removed and the SCSS following the linter rule?